docs/sphinx/using/examples/vqe.rst

```diff
@@ -0,0 +1,22 @@
+Variational Quantum Eigensolver
```

Comment:
Why does this page not contain a tab for Python/C++? There is a vqe_h2.cpp example to put here.

Reply:
I was trying to make all of these examples 1-to-1 and the only C++ VQE examples I could find on hand -- including vqe_h2.cpp -- use different models in their code. I'll link to that one here though and file an issue to write examples to match the python code.